docker/server/nginx/etc/nginx/conf.d/universe.marathon.mesos.conf

```diff
@@ -72,13 +72,25 @@ server {
     types {
       "application/vnd.dcos.universe.repo+json;charset=utf-8;version=v3" json;
     }
+    
+    if ($dcos_release_version = "1.10") {
+      rewrite ^/v3$ /repo-empty-v3.json break;
+    }
+    
+    
     rewrite ^/v3$ /repo-up-to-$dcos_release_version.json break;
   }
 
   location = /v4 {
     types {
       "application/vnd.dcos.universe.repo+json;charset=utf-8;version=v4" json;
     }
+    
+    if ($dcos_release_version != "1.10") {
+      rewrite ^/v4$ /repo-empty-v3.json break; # empty v3 = empty v4
```

Comment:
You're right, but it's likely not going to be worth doing everything you suggest, since we're planning on replacing it with universe-service soon. Let's just generate the empty v4 file for now.